src/components/CheckBox.js

```diff
@@ -5,6 +5,7 @@ import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes
 import Text from "./Text";
 import Touchable from "./Touchable";
 
+import checkboxUrl from "./check-box.png";
```

Comment:
We have a new convention to put assets in `/assets/images`. Do you want to move these 2 images?